rebasehelper/specfile.py

```diff
@@ -1110,6 +1113,24 @@ def get_prep_section(self):
                 result.append(prep.pop(0))
         return result
 
+    def get_main_files_section(self):
+        """Finds the exact name of the main files section.
+
+        Returns:
+            str: Name of the main files section.
+
+        """
+        parser = SilentArgumentParser()
+        parser.add_argument('-n', default=None)
+        parser.add_argument('-f')
+        parser.add_argument('section-name', nargs='?', default=None)
```

Comment:
This should be rather `subpackage-name`.

Reply:
And if you name it `subpackage_name` or simply `subpackage`, you don't have to use `getattr` to access it.